CMakeLists.txt

```diff
@@ -1,19 +1,53 @@
-cmake_minimum_required(VERSION 2.8)
-project(mustache)
-if (UNIX)
-	add_definitions(
-		-Wall
-		-Wextra
-		-Werror
-		-std=c++11
-	)
-elseif (MSVC)
-	add_definitions(
-		/W3
-		/WX
-	)
+cmake_minimum_required(VERSION 3.7.2)
```

Comment:
why 3.7.2? we should document why the version has changed in the cmake file.

Reply:
3.7.2 is the one shipped with debian stretch and widely used/tested
bumping requirement to 3.x allows using target based policies

Reply:
Ok, well AppVeyor tests passed, so seems fine.